index.js

```diff
@@ -24,20 +24,20 @@ module.exports = {
 // Similar to native `require` behavior, but doesn't check in `node_modules` folders
 // Based on https://github.com/js-cli/node-findup-sync
 function requireUp(filename, cwd) {
-  var filepath = path.resolve(cwd, filename);
+  var filepath = path.resolve(cwd, filename) + exts[i];
```

Comment:
@cletusw I missed this earlier, but `filepath` needs to be defined within the scope of the for loop so that it can access `i`

Reply:
In `v1.1.1`, the `filepath` variable evaluates to `/path/to/eslint-local-rulesundefined`.

This will cause MODULE_NOT_FOUND errors, an example might look like:
```
Error: Cannot find module '/path/to/eslint-local-rulesundefined'
```

Since this error message matches the pattern `Cannot find module '${filepath}'` that we check for, an error is not thrown, and this results in the standard 'Failed to load plugin' error being logged